cookie.md

```diff
@@ -0,0 +1,30 @@
+# Cookie
+
+**Prerequisite knowledge:** http, 
+
+## Intro
+An HTTP cookie is a piece of text that is transferred (both ways) between browser and server on every request. They were designed to handle the state of the application. Most common use-cases for cookies are Session handling and 3rd user party tracking.
```

Comment:
`cookie is a piece ... they were designed`. How about renaming the docs to `cookies` and use the plural form?